app/livechat/client/views/app/livechatCurrentChats.js

```diff
@@ -284,4 +296,11 @@ Template.livechatCurrentChats.onRendered(function() {
 		todayHighlight: true,
 		format: moment.localeData().longDateFormat('L').toLowerCase(),
 	});
+
+	// Set value of select
```

Comment:
Should use a helper to return the value to the template instead of set it manually